addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -2175,6 +2184,13 @@ const ListUserValueWidget = UserValueWidget.extend({
             stop: (event, ui) => {
                 this._notifyCurrentState();
             },
+            start: (event, ui) => {
+                // Avoid skipping the list rebuild after reordering.
```

Comment:
I'm not sure I understand this comment properly. Don't we skip the rebuild when it's a blur event? So by calling blur, we're skipping the list rebuild right?

Reply:
I will improve the comment but here is a little explanation:
Before doing this:
```js
if (this.listTable.contains(activeElement) && activeElement.tagName === 'INPUT') {
    activeElement.blur();
}
```
The input was blurred when it was dropped. With the change I'm introducing, the input will be blurred at the moment it is dragged. So here is a recap of what happen with my changes when a focused input is drag & dropped:
1. When the user starts drag, we blur the focused input, it set the `skipRebuildWeList` to `true`, it triggers a `notifyCurrentState` and then the `setValue` will be called but will not rebuild the list and will set `skipRebuildWeList` to `false`
2. When the user drop the list item, it will again `notifyCurrentState` and then a `setValue` wich will rebuild the list.

I hope my explanation is clear enough :sweat_smile:

Reply:
Comment adapted :heavy_check_mark: